spec/integration/aws_vpc_spec.rb

```diff
@@ -194,5 +194,25 @@
         end
       end
     end
+
+    with_aws "with no other objects" do
+      context "When operating in different regions" do
+        let(:regions) { %w(us-west-1 us-west-2) }
+
+        it "creates both VPCs" do
+          expect_recipe {
+            regions.each_with_index do |region, idx|
+              with_driver "aws::#{region}" do
```

Comment:
I don't think using `with_driver` works in a test because the driver is [being set](https://github.com/chef/chef-provisioning-aws/blob/fd47eecfb0246ac5bcd3f3844e13dfea385096c4/spec/aws_support.rb#L96) by the `AWS_TEST_DRIVER` environment variable